### ISSUE TITLE
v1.7.0 is not available on packagist

### ISSUE THREAD

**Issue by dimtrovich** [adhocore/php-cli#102]
![image](https://github.com/adhocore/php-cli/assets/37987162/d65c7144-f9ea-4dbb-ac5e-68ae19cc079d)

I'm trying to use the latest version of cli in my project, but it's not available on packagist.

![image](https://github.com/adhocore/php-cli/assets/37987162/59e587db-073c-489d-8ee4-e73112decb44)

I went to packagist and realized that indeed, this version is not listed on packagist although the release was published on github. @adhocore  , you must have forgotten to update packagist. 

![Capture d'écran 2024-04-05 161649](https://github.com/adhocore/php-cli/assets/37987162/2173884d-d2e5-42c1-a1c4-60268709e475)

**Comment by adhocore** [comment]
that should work automagically though

**Comment by dimtrovich** [comment]
in fact, in principle it should packagist should automatically support the new version, but it doesn't. 

I deleted "adhocore/cli" from the composer.json file and then ran "composer update" to remove it from the project.

I then ran "compose require adhocore/cli", which should install the latest version, but to my horror it installed version 1.6.2.

I don't know what could justify this behavior

![image](https://github.com/adhocore/php-cli/assets/37987162/52fc2fa0-17bd-442a-a468-aec5de352831)
